src/renderer/shared/hocs/withProgressChange.js

```diff
@@ -21,8 +21,10 @@ export default function withProgressChange(actions, progress, callback) {
   return (Component) => {
     class WrappedComponent extends React.Component {
       componentWillReceiveProps(nextProps) {
-        if (!progresses.includes(this.props[PROGRESS_PROP])
-            && progresses.includes(nextProps[PROGRESS_PROP])) {
+        if (
+          !progresses.includes(this.props[PROGRESS_PROP]) &&
```

Comment:
I'm not a big fan of this if statement xD

Reply:
@DalderupMaurice I reverted this, other than the `&&` placement.